BOM failure with 1.9.11

### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  <!-- Paste your environment details below -->
  ```text
  Paste the output here
  ```

</details>


### What Operating System are you using (both controller, and any agents involved in the problem)?

ci.jenkins.io

### Reproduction steps

When doing a [BOM run for 1.9.11](https://ci.jenkins.io/job/Tools/job/bom/view/change-requests/job/PR-6059/2/pipeline-overview/), there is a test failure.

### Expected Results

passing BOM tests

### Actual Results

```
12:08:22  ======= EXCEPTION START ========
12:08:22  Exception class=[org.htmlunit.corejs.javascript.EvaluatorException]
12:08:22  org.htmlunit.ScriptException: syntax error (http://localhost:41301/jenkins/adjuncts/31067dc8/com/dabsquared/gitlabjenkins/GitLabPushTrigger/adjunct.js#3)
12:08:22  	at org.htmlunit.javascript.JavaScriptEngine$HtmlUnitCompileContextAction.run(JavaScriptEngine.java:900)
12:08:22  	at org.htmlunit.corejs.javascript.Context.call(Context.java:638)
12:08:22  	at org.htmlunit.corejs.javascript.ContextFactory.call(ContextFactory.java:452)
12:08:22  	at org.htmlunit.javascript.HtmlUnitContextFactory.callSecured(HtmlUnitContextFactory.java:312)
12:08:22  	at org.htmlunit.javascript.JavaScriptEngine.compile(JavaScriptEngine.java:735)
12:08:22  	at org.htmlunit.javascript.JavaScriptEngine.compile(JavaScriptEngine.java:108)
12:08:22  	at org.htmlunit.html.HtmlPage.loadJavaScriptFromUrl(HtmlPage.java:1119)
12:08:22  	at org.htmlunit.html.HtmlPage.loadExternalJavaScriptFile(HtmlPage.java:1010)
12:08:22  	at org.htmlunit.html.ScriptElementSupport.executeScriptIfNeeded(ScriptElementSupport.java:185)
12:08:22  	at org.htmlunit.html.ScriptElementSupport$1.execute(ScriptElementSupport.java:112)
12:08:22  	at org.htmlunit.html.ScriptElementSupport.onAllChildrenAddedToPage(ScriptElementSupport.java:131)
12:08:22  	at org.htmlunit.html.HtmlScript.onAllChildrenAddedToPage(HtmlScript.java:216)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoDOMBuilder.endElement(HtmlUnitNekoDOMBuilder.java:526)
12:08:22  	at org.htmlunit.cyberneko.xerces.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:287)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoDOMBuilder.endElement(HtmlUnitNekoDOMBuilder.java:472)
12:08:22  	at org.htmlunit.cyberneko.HTMLTagBalancer.callEndElement(HTMLTagBalancer.java:1255)
12:08:22  	at org.htmlunit.cyberneko.HTMLTagBalancer.endElement(HTMLTagBalancer.java:1199)
12:08:22  	at org.htmlunit.cyberneko.filters.DefaultFilter.endElement(DefaultFilter.java:169)
12:08:22  	at org.htmlunit.cyberneko.filters.NamespaceBinder.endElement(NamespaceBinder.java:264)
12:08:22  	at org.htmlunit.cyberneko.HTMLScanner$ContentScanner.scanEndElement(HTMLScanner.java:3341)
12:08:22  	at org.htmlunit.cyberneko.HTMLScanner$ContentScanner.scan(HTMLScanner.java:2246)
12:08:22  	at org.htmlunit.cyberneko.HTMLScanner.scanDocument(HTMLScanner.java:965)
12:08:22  	at org.htmlunit.cyberneko.HTMLConfiguration.parse(HTMLConfiguration.java:355)
12:08:22  	at org.htmlunit.cyberneko.HTMLConfiguration.parse(HTMLConfiguration.java:313)
12:08:22  	at org.htmlunit.cyberneko.xerces.parsers.AbstractXMLDocumentParser.parse(AbstractXMLDocumentParser.java:79)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoDOMBuilder.parse(HtmlUnitNekoDOMBuilder.java:773)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoHtmlParser.parse(HtmlUnitNekoHtmlParser.java:179)
12:08:22  	at org.htmlunit.DefaultPageCreator.createHtmlPage(DefaultPageCreator.java:299)
12:08:22  	at org.htmlunit.DefaultPageCreator.createPage(DefaultPageCreator.java:218)
12:08:22  	at org.jvnet.hudson.test.HudsonPageCreator.createPage(HudsonPageCreator.java:53)
12:08:22  	at org.htmlunit.WebClient.loadWebResponseInto(WebClient.java:694)
12:08:22  	at org.htmlunit.WebClient.loadWebResponseInto(WebClient.java:587)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:504)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:413)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:550)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:532)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule$WebClient.goTo(JenkinsRule.java:2729)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule$WebClient.goTo(JenkinsRule.java:2708)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule$WebClient.getPage(JenkinsRule.java:2663)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule.configRoundtrip(JenkinsRule.java:1387)
12:08:22  	at org.jenkinsci.plugins.workflow.steps.StepConfigTester.configRoundTrip(StepConfigTester.java:61)
12:08:22  	at io.jenkins.tools.bom.sample.ExampleStepTest.smokes(ExampleStepTest.java:38)
12:08:22  Caused by: org.htmlunit.corejs.javascript.EvaluatorException: syntax error (http://localhost:41301/jenkins/adjuncts/31067dc8/com/dabsquared/gitlabjenkins/GitLabPushTrigger/adjunct.js#3)
12:08:22  	at org.htmlunit.javascript.HtmlUnitContextFactory$HtmlUnitErrorReporter.error(HtmlUnitContextFactory.java:388)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addError(Parser.java:247)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.reportError(Parser.java:327)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.reportError(Parser.java:319)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.reportError(Parser.java:315)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3492)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.arrayLiteral(Parser.java:3606)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3384)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2563)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expr(Parser.java:2519)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nameOrLabel(Parser.java:2267)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statementHelper(Parser.java:1410)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statement(Parser.java:1262)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parseFunctionBody(Parser.java:736)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:1012)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:937)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3380)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2563)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expr(Parser.java:2519)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nameOrLabel(Parser.java:2267)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statementHelper(Parser.java:1410)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statement(Parser.java:1262)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parseFunctionBody(Parser.java:736)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:1012)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:937)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3380)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.argumentList(Parser.java:2939)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.makeFunctionCall(Parser.java:3092)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExprTail(Parser.java:3062)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:3003)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expr(Parser.java:2519)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nameOrLabel(Parser.java:2267)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statementHelper(Parser.java:1410)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statement(Parser.java:1262)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parse(Parser.java:636)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parse(Parser.java:564)
12:08:22  	at org.htmlunit.corejs.javascript.Context.parse(Context.java:2649)
12:08:22  	at org.htmlunit.corejs.javascript.Context.compileImpl(Context.java:2584)
12:08:22  	at org.htmlunit.corejs.javascript.Context.compileString(Context.java:1529)
12:08:22  	at org.htmlunit.javascript.HtmlUnitContextFactory$TimeoutContext.compileString(HtmlUnitContextFactory.java:195)
12:08:22  	at org.htmlunit.corejs.javascript.Context.compileString(Context.java:1517)
12:08:22  	at org.htmlunit.javascript.JavaScriptEngine$HtmlUnitCompileContextAction.run(JavaScriptEngine.java:893)
12:08:22  	... 41 more
12:08:22  Enclosed exception: 
12:08:22  org.htmlunit.corejs.javascript.EvaluatorException: syntax error (http://localhost:41301/jenkins/adjuncts/31067dc8/com/dabsquared/gitlabjenkins/GitLabPushTrigger/adjunct.js#3)
12:08:22  	at org.htmlunit.javascript.HtmlUnitContextFactory$HtmlUnitErrorReporter.error(HtmlUnitContextFactory.java:388)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addError(Parser.java:247)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.reportError(Parser.java:327)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.reportError(Parser.java:319)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.reportError(Parser.java:315)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3492)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.arrayLiteral(Parser.java:3606)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3384)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2563)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expr(Parser.java:2519)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nameOrLabel(Parser.java:2267)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statementHelper(Parser.java:1410)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statement(Parser.java:1262)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parseFunctionBody(Parser.java:736)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:1012)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:937)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3380)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2563)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expr(Parser.java:2519)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nameOrLabel(Parser.java:2267)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statementHelper(Parser.java:1410)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statement(Parser.java:1262)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parseFunctionBody(Parser.java:736)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:1012)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.function(Parser.java:937)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.primaryExpr(Parser.java:3380)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:2968)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.argumentList(Parser.java:2939)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.makeFunctionCall(Parser.java:3092)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExprTail(Parser.java:3062)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.memberExpr(Parser.java:3003)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.unaryExpr(Parser.java:2871)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expExpr(Parser.java:2781)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.mulExpr(Parser.java:2764)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.addExpr(Parser.java:2750)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.shiftExpr(Parser.java:2733)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.relExpr(Parser.java:2711)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.eqExpr(Parser.java:2687)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitAndExpr(Parser.java:2678)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitXorExpr(Parser.java:2669)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.bitOrExpr(Parser.java:2660)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.andExpr(Parser.java:2651)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.orExpr(Parser.java:2642)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nullishCoalescingExpr(Parser.java:2623)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.condExpr(Parser.java:2591)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.assignExpr(Parser.java:2545)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.expr(Parser.java:2519)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.nameOrLabel(Parser.java:2267)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statementHelper(Parser.java:1410)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.statement(Parser.java:1262)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parse(Parser.java:636)
12:08:22  	at org.htmlunit.corejs.javascript.Parser.parse(Parser.java:564)
12:08:22  	at org.htmlunit.corejs.javascript.Context.parse(Context.java:2649)
12:08:22  	at org.htmlunit.corejs.javascript.Context.compileImpl(Context.java:2584)
12:08:22  	at org.htmlunit.corejs.javascript.Context.compileString(Context.java:1529)
12:08:22  	at org.htmlunit.javascript.HtmlUnitContextFactory$TimeoutContext.compileString(HtmlUnitContextFactory.java:195)
12:08:22  	at org.htmlunit.corejs.javascript.Context.compileString(Context.java:1517)
12:08:22  	at org.htmlunit.javascript.JavaScriptEngine$HtmlUnitCompileContextAction.run(JavaScriptEngine.java:893)
12:08:22  	at org.htmlunit.corejs.javascript.Context.call(Context.java:638)
12:08:22  	at org.htmlunit.corejs.javascript.ContextFactory.call(ContextFactory.java:452)
12:08:22  	at org.htmlunit.javascript.HtmlUnitContextFactory.callSecured(HtmlUnitContextFactory.java:312)
12:08:22  	at org.htmlunit.javascript.JavaScriptEngine.compile(JavaScriptEngine.java:735)
12:08:22  	at org.htmlunit.javascript.JavaScriptEngine.compile(JavaScriptEngine.java:108)
12:08:22  	at org.htmlunit.html.HtmlPage.loadJavaScriptFromUrl(HtmlPage.java:1119)
12:08:22  	at org.htmlunit.html.HtmlPage.loadExternalJavaScriptFile(HtmlPage.java:1010)
12:08:22  	at org.htmlunit.html.ScriptElementSupport.executeScriptIfNeeded(ScriptElementSupport.java:185)
12:08:22  	at org.htmlunit.html.ScriptElementSupport$1.execute(ScriptElementSupport.java:112)
12:08:22  	at org.htmlunit.html.ScriptElementSupport.onAllChildrenAddedToPage(ScriptElementSupport.java:131)
12:08:22  	at org.htmlunit.html.HtmlScript.onAllChildrenAddedToPage(HtmlScript.java:216)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoDOMBuilder.endElement(HtmlUnitNekoDOMBuilder.java:526)
12:08:22  	at org.htmlunit.cyberneko.xerces.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:287)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoDOMBuilder.endElement(HtmlUnitNekoDOMBuilder.java:472)
12:08:22  	at org.htmlunit.cyberneko.HTMLTagBalancer.callEndElement(HTMLTagBalancer.java:1255)
12:08:22  	at org.htmlunit.cyberneko.HTMLTagBalancer.endElement(HTMLTagBalancer.java:1199)
12:08:22  	at org.htmlunit.cyberneko.filters.DefaultFilter.endElement(DefaultFilter.java:169)
12:08:22  	at org.htmlunit.cyberneko.filters.NamespaceBinder.endElement(NamespaceBinder.java:264)
12:08:22  	at org.htmlunit.cyberneko.HTMLScanner$ContentScanner.scanEndElement(HTMLScanner.java:3341)
12:08:22  	at org.htmlunit.cyberneko.HTMLScanner$ContentScanner.scan(HTMLScanner.java:2246)
12:08:22  	at org.htmlunit.cyberneko.HTMLScanner.scanDocument(HTMLScanner.java:965)
12:08:22  	at org.htmlunit.cyberneko.HTMLConfiguration.parse(HTMLConfiguration.java:355)
12:08:22  	at org.htmlunit.cyberneko.HTMLConfiguration.parse(HTMLConfiguration.java:313)
12:08:22  	at org.htmlunit.cyberneko.xerces.parsers.AbstractXMLDocumentParser.parse(AbstractXMLDocumentParser.java:79)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoDOMBuilder.parse(HtmlUnitNekoDOMBuilder.java:773)
12:08:22  	at org.htmlunit.html.parser.neko.HtmlUnitNekoHtmlParser.parse(HtmlUnitNekoHtmlParser.java:179)
12:08:22  	at org.htmlunit.DefaultPageCreator.createHtmlPage(DefaultPageCreator.java:299)
12:08:22  	at org.htmlunit.DefaultPageCreator.createPage(DefaultPageCreator.java:218)
12:08:22  	at org.jvnet.hudson.test.HudsonPageCreator.createPage(HudsonPageCreator.java:53)
12:08:22  	at org.htmlunit.WebClient.loadWebResponseInto(WebClient.java:694)
12:08:22  	at org.htmlunit.WebClient.loadWebResponseInto(WebClient.java:587)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:504)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:413)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:550)
12:08:22  	at org.htmlunit.WebClient.getPage(WebClient.java:532)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule$WebClient.goTo(JenkinsRule.java:2729)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule$WebClient.goTo(JenkinsRule.java:2708)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule$WebClient.getPage(JenkinsRule.java:2663)
12:08:22  	at org.jvnet.hudson.test.JenkinsRule.configRoundtrip(JenkinsRule.java:1387)
12:08:22  	at org.jenkinsci.plugins.workflow.steps.StepConfigTester.configRoundTrip(StepConfigTester.java:61)
12:08:22  	at io.jenkins.tools.bom.sample.ExampleStepTest.smokes(ExampleStepTest.java:38)
12:08:22  == CALLING JAVASCRIPT ==
12:08:22  Behaviour.specify("BUTTON.gitlab_plugin-generate", "gitlab_plugin-generate", 0, function (e) {
12:08:22      e.onclick = function (evt) {
12:08:22          document.getElementById('gitlab_plugin_secretToken').value = [...Array(32)].map(() => Math.floor(Math.random() * 16).toString(16)).join('');
12:08:22          evt.preventDefault();
12:08:22      };
12:08:22  });
12:08:22  
12:08:22  Behaviour.specify("BUTTON.gitlab_plugin-clear", "gitlab_plugin-clear", 0, function (e) {
12:08:22      e.onclick = function (evt) {
12:08:22          document.getElementById('gitlab_plugin_secretToken').value = "";
12:08:22          evt.preventDefault();
12:08:22      };
12:08:22  });
12:08:22  
12:08:22  ======= EXCEPTION END ========
12:08:22  
12:08:25  [INFO] Running io.jenkins.tools.bom.sample.InjectedTest
12:08:52  [INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 26.68 s -- in io.jenkins.tools.bom.sample.InjectedTest
12:08:52  [INFO] 
12:08:52  [INFO] Results:
12:08:52  [INFO] 
12:08:52  [ERROR] Errors: 
12:08:52  [ERROR]   ExampleStepTest.smokes:38 » Script syntax error (http://localhost:41301/jenkins/adjuncts/31067dc8/com/dabsquared/gitlabjenkins/GitLabPushTrigger/adjunct.js#3)
12:08:52  [INFO] 
12:08:52  [ERROR] Tests run: 8, Failures: 0, Errors: 1, Skipped: 0
```

### Anything else?

_No response_

### Are you interested in contributing a fix?

_No response_

--- a/src/main/resources/com/dabsquared/gitlabjenkins/GitLabPushTrigger/adjunct.js
+++ b/src/main/resources/com/dabsquared/gitlabjenkins/GitLabPushTrigger/adjunct.js
@@ -1,6 +1,8 @@
 Behaviour.specify("BUTTON.gitlab_plugin-generate", "gitlab_plugin-generate", 0, function (e) {
     e.onclick = function (evt) {
-        document.getElementById('gitlab_plugin_secretToken').value = [...Array(32)].map(() => Math.floor(Math.random() * 16).toString(16)).join('');
+        const array = new Uint32Array(4);
+        self.crypto.getRandomValues(array);
+        document.getElementById('gitlab_plugin_secretToken').value = Array.from(array).map(e => e.toString(16)).join("");
         evt.preventDefault();
     };
 });

--- a/src/test/java/com/dabsquared/gitlabjenkins/util/JavaScriptTest.java
+++ b/src/test/java/com/dabsquared/gitlabjenkins/util/JavaScriptTest.java
@@ -1,0 +1,26 @@
+package com.dabsquared.gitlabjenkins.util;
+
+import com.dabsquared.gitlabjenkins.GitLabPushTrigger;
+import hudson.model.FreeStyleProject;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+import org.xml.sax.SAXException;
+
+/**
+ * JS in HTMLUnit isn't great, so ensure our JS doesn't cause it to fall over.
+ */
+@WithJenkins
+public class JavaScriptTest {
+    @Test
+    void testJavaScriptExecution(JenkinsRule j) throws IOException, SAXException {
+        final FreeStyleProject fs = j.createFreeStyleProject();
+        final GitLabPushTrigger trigger = new GitLabPushTrigger();
+        fs.addTrigger(trigger);
+        fs.save();
+        try (JenkinsRule.WebClient wc = j.createWebClient()) {
+            wc.getPage(fs, "configure");
+        }
+    }
+}

